Dockerfile

```diff
@@ -4,36 +4,34 @@ MAINTAINER unum@cloud.com
 
 WORKDIR ./
 
-RUN apt-get update
+RUN apt update
 ENV DEBIAN_FRONTEND noninteractive
 
 # Install tools
 RUN apt install -y python3-pip
 RUN pip3 install cmake
-RUN pip3 install conan
 RUN apt install -y gcc-10
 RUN apt install -y g++-10
-RUN apt-get install -y libexplain-dev
-RUN apt-get install -y libsnappy-dev
-RUN apt-get install -yq pkg-config
-RUN apt-get install -y git
+RUN apt install -y libexplain-dev
+RUN apt install -y libsnappy-dev
+RUN apt install -y pkg-config
+RUN apt install -y git
 
 # Build WiredTiger (latest)
-RUN git clone git://github.com/wiredtiger/wiredtiger.git
+RUN git clone  --depth 1 git@github.com:wiredtiger/wiredtiger.git
```

Comment:
I like the `--depth 1` added here, maybe we should add it in other `git clone`-s in this file?